definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -132,29 +106,5 @@
     "Label": "Payment Method: Cheque",
     "FieldType": "Label",
     "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "12/08/2020",
-    "CaseTypeID": "DIVORCE",
-    "ID": "PBAReferenceLabel",
-    "Label": "Your pay account reference: ${PBAReference}",
-    "FieldType": "Label",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "12/08/2020",
```

Comment:
```
  {
    "LiveFrom": "01/01/2017",
    "CaseTypeID": "DIVORCE",
    "ID": "LabelSolicitorFeeAccountNumber",
    "Label": "Fee account number: **${SolicitorFeeAccountNumber}**",
    "FieldType": "Label",
    "SecurityClassification": "Public"
  },
```

Reply:
I will not use it as it has slightly different label and I want to make sure it's exactly the same as provided mocks.